### DISCUSSION TITLE
Is it possible that weird cpu usage is an issue? It seems that DATANODE cannot properly utilize multiple cpu

### DISCUSSION THREAD

**Discussion by JavaCPlusMario** [apache/iotdb#13541] [Q&A]
# atop 
CPU | sys      32%  | user    485%
cpu | sys       4%  | user     67%
cpu | sys       4%  | user     63% 
...

    PID     TID S  CPU COMMAND-LINE (horizontal scroll with <- and -> keys)                                   
  39738       - S 533% /usr/java/latest/bin/java --add-opens=java.base/java.util.concurrent=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.nio=ALL-UNNAMED --add

It seems that DATANODE cannot properly utilize multiple cpus

**Comment by CritasWang** [comment]
jdk version?
iotdb version?
